package-lock.json

```diff
@@ -36,7 +36,7 @@
       "devDependencies": {
         "@edx/frontend-build": "^11.0.2",
         "@edx/frontend-platform": "2.4.0",
-        "@edx/paragon": "^20.21.3",
+        "@edx/paragon": "^20.27.0",
```

Comment:
Just to have it mentioned in the PR, there will be a PR updating that library in the course-authoring as well